00new_packages.autobuild

```diff
@@ -13,5 +13,5 @@ end
 
 cmake_package 'gui/rock_webapp'
 ruby_package 'tools/rest_api' do |pkg|
-    Autoproj.env_set 'ROCK_WEBAPP', File.join(pkg.prefix, 'share', 'rest_api')
+    Autoproj.env_set 'ROCK_WEBAPP', File.join(Autoproj.build_dir, 'share', 'rest_api')
```

Comment:
If you don't have the time to change ROCK_WEBAPP into a path, then set the envvar into `gui/rock_webapp`  to `pkg.prefix`. And everyone should do the same if they expect to export a webapp "configuration directory".

It will work in cases where everyone is installed in the same prefix.

It will make autoproj fail if two packages try to set ROCK_WEBAPP in two different places.